src/math/p5.Vector.js

```diff
@@ -658,24 +702,39 @@ p5.Vector.prototype.mult = function mult(n) {
  * </code>
  * </div>
  */
-p5.Vector.prototype.div = function div(n) {
-  if (!(typeof n === 'number' && isFinite(n))) {
-    console.warn(
-      'p5.Vector.prototype.div:',
-      'n is undefined or not a finite number'
-    );
+p5.Vector.prototype.div = function div(x, y, z) {
+  if (x instanceof p5.Vector) {
+    if (x.x === 0 || x.y === 0 || x.z === 0) {
+      console.warn('p5.Vector.prototype.div:', 'divide by 0');
+      return this;
+    }
+    this.x /= x.x || 1;
+    this.y /= x.y || 1;
+    this.z /= x.z || 1;
```

Comment:
Same with above regarding default case

Reply:
In this case, it's probably best to not have the default be 0, since this would cause a divide by zero.